code/stable/glassbr/src/cpp/designLog.txt

```diff
@@ -0,0 +1,24 @@
+Languages selected as Python Cpp CSharp Java Swift..
```

Comment:
Shouldn't the language say cpp?  The other languages have their own folders, where the selected language would correspond to the folder.